modules/osd-applications-config-custom-domains.adoc

```diff
@@ -122,3 +122,6 @@ $ oc get route -n my-project
 $ curl https://hello-openshift-tls-my-project.apps.<company_name>.io
 Hello OpenShift!
 ----
+
+.Troubleshooting
+* link:https://access.redhat.com/solutions/5419501[Error creating TLS secret]
```

Comment:
In build: https://62748--docspreview.netlify.app/openshift-rosa/latest/applications/deployments/osd-config-custom-domains-applications.html